### PR TITLE
This property of MSBuild support by C# too

### PR DESCRIPTION
**Pull request by kant2002**
GenerateDocumentation supported by VB and C# definitely.
See https://github.com/dotnet/sdk/blob/b28a8efbfb30d84019351abb3aa9a2b15e008662/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.BeforeCommon.targets#L178

### CODE REVIEW

**Comment by PRMerger20** [comment]
@kant2002 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Mikejo5000** [comment]
@rainersigwald to review. Do we need to specify whether it also works in C++ projects?

**Comment by rainersigwald** [comment]
Yeah, I think "doesn't apply to C++ projects" would be good here. It's definitely not VB only!

File: docs/msbuild/common-msbuild-project-properties.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ The following table lists frequently used properties that are defined in the Vis
...
 | FileAlignment | Specifies, in bytes, where to align the sections of the output file. Valid values are 512, 1024, 2048, 4096, 8192. This property is equivalent to the `/filealignment` compiler switch. |
 | FrameworkPathOverride | Specifies the location of *mscorlib.dll* and *microsoft.visualbasic.dll*. This parameter is equivalent to the `/sdkpath` switch of the *vbc.exe* compiler. |
-| GenerateDocumentation | (Visual Basic only) A boolean parameter that indicates whether documentation is generated by the build. If `true`, the build generates documentation information and puts it in an *.xml* file together with the name of the executable file or library that the build task created. |
+| GenerateDocumentation | A boolean parameter that indicates whether documentation is generated by the build. If `true`, the build generates documentation information and puts it in an *.xml* file together with the name of the executable file or library that the build task created. |
```
Comment (Mikejo5000) [inline docs/msbuild/common-msbuild-project-properties.md]:
```suggestion
| GenerateDocumentation | (C#, Visual Basic) A boolean parameter that indicates whether documentation is generated by the build. If `true`, the build generates documentation information and puts it in an *.xml* file together with the name of the executable file or library that the build task created. |
```

Comment (Mikejo5000) [reply docs/msbuild/common-msbuild-project-properties.md]: @kant2002 Thanks for the PR! I added a suggestion to clarify.

**Comment by Mikejo5000** [comment]
#sign-off

**Comment by Mikejo5000** [comment]
@kant2002 Thanks!

**Pull request merged**